pokemongo_bot/cell_workers/drop_items_worker.py

```diff
@@ -0,0 +1,135 @@
+import random
+
+from pokemongo_bot.human_behaviour import sleep
+from pokemongo_bot.item_list import Item
+from pokemongo_bot import logger
+
+class DropItemsWorker(object):
+    def __init__(self, bot, items_count=0, max_storage=100, items={}):
+        self.bot = bot
+        self.api = bot.api
+        self.items_count = items_count
+        self.max_storage = max_storage
+        self.items = items
+
+    def check(self):
```

Comment:
The entry points to all workers needs to be called `work`. You can have your work function call something else though if you'd like.